crates/humanode-peer/src/chain_spec.rs

```diff
@@ -252,6 +252,35 @@ const EXISTENTIAL_DEPOSIT: Balance = 500;
 /// The initial pot accounts balance for testnet genesis.
 const INITIAL_POT_ACCOUNT_BALANCE: Balance = EXISTENTIAL_DEPOSIT + DEV_ACCOUNT_BALANCE;
 
+/// A helper function to calculate bridge pot balances.
+#[allow(clippy::type_complexity)]
```

Comment:
I don't like this, definitely not justified

Reply:
Forgot to remove after refactoring, thanks.

Reply:
https://github.com/humanode-network/humanode/pull/700/commits/d58e2871599e3dbeffb7cbdb1363acdca0ba8e39